_docs/templates/email-template.md

```diff
@@ -211,56 +278,75 @@ the user to scan for pertinent information quickly.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Call Out box used for workarounds and comments. Comment includes code block."] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Comment%20with%20CodeBlock.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Comment%20with%20CodeBlock.svg)
 {% endfigure %}
 {% endcolumn %}
 </div>
 
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol-md-4" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
 -   **Code Blocks** - show user code in a human readable format. Either as part
   of a workaround or just to display system content.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Code block visual examples and specifications"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Code%20Block%20.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Code%20Block%20.svg)
 {% endfigure %}
 {% endcolumn %}
 </div>
 
+
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
 ### Buttons
 
 Buttons should only be used as a gateway to control panels to view more
 detailed information or respond to a ticket.
 
+{% endcolumn %}
+</div>
+
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol-md-4" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
 They should never break up the content of an email, and there should never be
 more than one button in the email. When adding buttons only use the standard
 button styles, and text should be in all caps. Go to the [Buttons page](http://design.rax.io/design-system/components/buttons.html)
```

Comment:
Update link to `[Buttons page]({{site.url}}/components/buttons.html)`.

Reply:
updated.